pkg/controller/certificates/sync.go

```diff
@@ -57,10 +58,19 @@ const (
 	messageIssuingCertificate  = "Issuing certificate..."
 	messageRenewingCertificate = "Renewing certificate..."
 
+	infoCertificateDuration = "WarnCertificateDuration"
+	infoScheduleModified    = "WarnScheduleModified"
+
 	messageCertificateIssued  = "Certificate issued successfully"
 	messageCertificateRenewed = "Certificate renewed successfully"
+
+	messageCertificateDuration = "Certificate received from server has a validity duration of %s. The requested certificate validity duration was %s"
+	messageScheduleModified    = "Certificate renewal duration was changed to fit inside the received certificate validity duration from issuer."
 )
 
+// to help testing
+var now = time.Now
```

Comment:
I kind of hate this line, and I don't understand it's purpose in the original commit. I know that it is used in the test (next file down), but there must be a better way to handle it.